api/server/router/swarm/cluster_routes.go

```diff
@@ -178,15 +187,20 @@ func (sr *swarmRouter) getServices(ctx context.Context, w http.ResponseWriter, r
 
 func (sr *swarmRouter) getService(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 	var insertDefaults bool
+
 	if value := r.URL.Query().Get("insertDefaults"); value != "" {
 		var err error
 		insertDefaults, err = strconv.ParseBool(value)
 		if err != nil {
-			err := fmt.Errorf("invalid value for insertDefaults: %s", value)
 			return errors.Wrapf(errdefs.InvalidParameter(err), "invalid value for insertDefaults: %s", value)
 		}
 	}
 
+	// the Status field is not supported when retrieving an individual service
+	// because the Backend API changes required to accomodate it would be too
+	// disruptive, and because that field is so rarely needed as part of an
+	// individual service inspection.
```

Comment:
Position of this comment is a bit confusing (as there is no code visible here for `status`?)